src/main/interface/cli.c

```diff
@@ -3239,10 +3262,54 @@ static void cliDumpRateProfile(uint8_t rateProfileIndex, uint8_t dumpMask)
     rateProfileIndexToUse = CURRENT_PROFILE_INDEX;
 }
 
+#ifdef USE_CLI_BATCH
+static void cliPrintCommandBatchWarning(const char *warning)
+{
+    cliPrintWarningLinef("Errors were detected - please review before continuing");
+    if (warning) {
+        cliPrintWarningLinef(warning);
+    }
+}
+
+static void resetCommandBatch(void)
+{
+    commandBatchActive = false;
+    commandBatchError = false;
+}
+
+static void cliBatch(char *cmdline)
+{
+    if (strncasecmp(cmdline, "start", 5) == 0) {
```

Comment:
I don't like N component as it will allow substring match, not whole match.
E.g. `batch starting` will also pass the check